.npmignore

```diff
@@ -1,13 +1,6 @@
-.awcache
-.vscode
-.env*
-.storybook
-.fusebox
-.babelrc
-
-externals
-src
-__mocks__
-reaction-force.js
-reaction-force.js.map
-gulpfile.js
\ No newline at end of file
+# Ignore everything except what's explicitly whitelisted
```

Comment:
So much negation in this file... isn’t this including all files and then ignore some?

Reply:
Nope. This is an ignore file. So the `*` is ignore everything. The `!` just says don't ignore this thing.

Reply:
So it's like... ignore everything. Don't ignore dist stuff. But _do_ ignore __mocks__, __tests__, and __stories__ in dist.

Reply:
Yeah sorry, I meant to comment on the `!dist/**` part. It basically includes everything first and then ignores some again.